src/Components/BookCard/BookCard.js

```diff
@@ -12,12 +13,23 @@ const BookCard = ({ bookInfo, addToToReadList, addToHaveReadList } ) => {
           <h5 className='rank'>Rank: {bookInfo.rank}</h5>
         </div>
         <div className='book-btns'>
-          <button className='to-read-btn' type='button' value={bookInfo.title} onClick={event => addToToReadList(event)}>To Read</button>
-          <button className='have-read-btn' type='button' value={bookInfo.title} onClick={event => addToHaveReadList(event)}>Have Read</button>
+          {!toReadList.includes(bookInfo) &&
```

Comment:
definitely something to add as a future issue, when we are not concerned about the time constraints. Either just having the array reject duplicates, or having the buttons only render if the book is not in a list. Adding a Boolean in state to indicate if a book is read, or is a favorite might be a good solution to the button rendering issue